Research/ResearchUI/iOS/Step View Controllers/RSDScrollingOverviewStepViewController.swift

```diff
@@ -141,58 +139,279 @@ open class RSDScrollingOverviewStepViewController: RSDOverviewStepViewController
         return Bundle(for: RSDScrollingOverviewStepViewController.self)
     }
+}
+
+/// `RSDTitleHeaderCollectionViewHeader` shows a simple title label.
+public class RSDTitleHeaderCollectionViewHeader: RSDCollectionViewCell {
+    
+    static let kCollectionHeaderTopMargin: CGFloat = 8.0
+    static let kCollectionHeaderBottomMargin: CGFloat = 16.0
+    
+    @IBOutlet public var titleLabel: UILabel?
+
+    fileprivate static var titleTextType: RSDDesignSystem.TextType = .mediumHeader
+
+    public override init(frame: CGRect) {
+        super.init(frame:frame)
+        commonInit()
+    }
 
+    public required init?(coder aDecoder: NSCoder) {
+        super.init(coder:aDecoder)
+        commonInit()
+    }
+
+    func updateColorsAndFonts() {
+        let designSystem = self.designSystem ?? RSDDesignSystem()
+        let background = self.backgroundColorTile ?? RSDGrayScale().white
+        let contentTile = designSystem.colorRules.tableCellBackground(on: background, isSelected: isSelected)
+
+        contentView.backgroundColor = contentTile.color
+        titleLabel?.textColor = designSystem.colorRules.textColor(on: contentTile, for: RSDTitleHeaderCollectionViewHeader.titleTextType)
+        titleLabel?.font = RSDTitleHeaderCollectionViewHeader.titleLabelFont(for: designSystem)
+    }
+    
+    fileprivate static func titleLabelFont(for designSystem: RSDDesignSystem) -> UIFont {
+        return designSystem.fontRules.font(for: RSDTitleHeaderCollectionViewHeader.titleTextType)
+    }
+
+    override open func setDesignSystem(_ designSystem: RSDDesignSystem, with background: RSDColorTile) {
+        super.setDesignSystem(designSystem, with: background)
+        updateColorsAndFonts()
+    }
+
+    private func commonInit() {
+        // Add the title label.
+        titleLabel = RSDTitleHeaderCollectionViewHeader.createTitleLabel()
+        contentView.addSubview(titleLabel!)
+
+        titleLabel!.translatesAutoresizingMaskIntoConstraints = false
+        titleLabel!.rsd_alignCenterHorizontal(padding: 0.0)
+        titleLabel!.rsd_alignToSuperview([.top], padding: RSDTitleHeaderCollectionViewHeader.kCollectionHeaderTopMargin)
+        titleLabel!.rsd_alignToSuperview([.bottom], padding: RSDTitleHeaderCollectionViewHeader.kCollectionHeaderBottomMargin)
+
+        updateColorsAndFonts()
+        setNeedsUpdateConstraints()
+    }
+    
+    fileprivate static func createTitleLabel() -> UILabel {
+        let label = UILabel()
+        label.numberOfLines = 1
+        label.textAlignment = .center
+        return label
+    }
+}
+
+/// `RSDOverviewCollectionViewCell` shows a vertically stacked image icon and title label.
+@IBDesignable open class RSDOverviewCollectionViewCell: RSDCollectionViewCell {
+
+    let kCollectionCellVeritcalItemSpacing = CGFloat(6)
+    
+    @IBOutlet public var titleLabel: UILabel?
+    @IBOutlet public var imageView: UIImageView?
+
+    open private(set) var titleTextType: RSDDesignSystem.TextType = .small
```

Comment:
Should not be using a dynamic font here. Real-estate is probably more important that accessibility though you should check with Stockard.

Reply:
This is copied straight from RSDTableSectionHeader so I figured it was correct?

Reply:
Table sections should dynamically resize. The title label here is a *caption* and those should stay small b/c real-estate is more important than readability. For someone who can't read this, they can turn on voiceover.

Typically, a user who uses the dynamic text sizing to make it extra big is use to using voiceover for certain elements or they put on their reading glasses. Instead, it would be much more useful here to set the `accessibilityLabel` for the cell as the caption and set the text to static.